docs/questions/visualizations/progress-bar.md

```diff
@@ -18,23 +18,87 @@ Progress bars give you an option to set up an alert whenever the result of a que
 
 ## How to create a progress bar
 
-To create a progress bar you'll need:
+### Data shape for the progress bar
 
-- A query that returns a single number, like "Sum of order quantity". Progress bar doesn't work with breakouts.
+To create a progress bar you'll need:
 
-  You can also use a query that returns several metrics in a single row:
+- A query that returns a single row with one or more numeric columns, like "Sum of order quantity". Progress bar doesn't work with breakouts.
 
   | Sum of Quantity | Average Quantity | Max Quantity |
   | --------------- | ---------------- | ------------ |
   | 4910            | 17.32            | 173          |
 
-In this case, the progress bar chart will use the metric in the first column ("Sum of quantity"). To change the metric, go to the editor and drag the metric you want to the first position in the Summarize block.
-
-- A goal value. The goal value should be a positive number. Currently, Metabase only supports setting a static goal (you can't set a goal based on another query).
+- A goal value. The goal value can be a positive number or a value from another column in the same query, see [Set progress bar goal](#set-progress-bar-goal).
 
   The goal is set in the [chart options](#progress-bar-options).
 
-![Progress bar KPI](../images/progress-bar-elements.png)
+  ![Progress bar KPI](../images/progress-bar-elements.png)
+
+### Create a progress bar
```

Comment:
adding this because couple of days ago someone complained that the article doesn't actually say how to create a viz